src/workloads/issues/ConnectionPoolStress.yml

```diff
@@ -0,0 +1,125 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload was created to reproduce issues discovered during sharded stress testing. The workload can cause
+  low performance, spikes in latency and excessive connection creation (on mongos processes). The metrics to look at
+  for this workload are the latencies and more specifically the Latency90thPercentile to Latency99thPercentile metrics.
```

Comment:
Does this have to run sharded?

Reply:
> Did the stability/variability measures turn up anything?
The issue is on the mongos and the stability metrics are only calculated for [mongod's](https://github.com/10gen/dsi/blob/45ead140cd76a8c0ec6652ec3115a3b88ec531bd/configurations/analysis/analysis.common.yml#L168-L169). 

It might be worth a patch to see if they can be calculated.

> This existing text is good. Could you add additional description of what the workload actually does?
Sure thing.

> Does this have to run sharded? 

It fails due to the ShardingTaskExecutorPoolMaxSize command. I'll look at ignoring the error and see what happens.